stable/enterprise/tests/common_helpers_test.yaml

```diff
@@ -1034,4 +1034,52 @@ tests:
                 - pod-template-hash
               maxSkew: 1
               topologyKey: kubernetes.io/hostname
-              whenUnsatisfiable: DoNotSchedule
\ No newline at end of file
+              whenUnsatisfiable: DoNotSchedule
+
+  - it: should render enterprise image when passed in as a string
+    templates: *backend_test_templates
+    documentIndex: 0
+    set:
+      image: "anchore/enterprise:9.9.9"
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[0].image
+          value: "anchore/enterprise:9.9.9"
+
+
+  - it: should render enterprise image when passed in as a string
+    templates: *backend_test_templates
+    documentIndex: 0
+    set:
+      image: "anchore/enterprise:9.9.9"
+    asserts:
+      - equal:
+          path: spec.template.spec.containers[0].image
+          value: "anchore/enterprise:9.9.9"
```

Comment:
Seems to be a duplicate?

Reply:
yep. you are correct. lol. one of them should have tested enterprise-ui. 🤦 ; I'll update it in a subsequent PR. Thanks @bot190 !